lib/ansible/module_utils/facts.py

```diff
@@ -1943,6 +1944,35 @@ def get_dmi_facts(self):
         rc, out, err = self.module.run_command("/usr/sbin/lsattr -El sys0 -a fwversion")
         data = out.split()
         self.facts['firmware_version'] = data[1].strip('IBM,')
+        rc, out, err = self.module.run_command("/usr/sbin/lsconf")
+        if out:
```

Comment:
i would condition to `rc == 0` unless the tool is known to violate that standard.